### PR TITLE
Feat/split v2 indexes

### PR DESCRIPTION
**Pull request by carpawell**
Blocked by https://github.com/nspcc-dev/neofs-sdk-go/pull/575

### CODE REVIEW

File: pkg/local_object_storage/metabase/VERSION.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ This file describes changes between the metabase versions.
 ## Current
 
 Numbers stand for a single byte value.
+The lowest not used bucket index: 20.
```
Comment (cthulhu-rider) [inline pkg/local_object_storage/metabase/VERSION.md:8]: incrementing this is one more thing to support and remember, dont think it's worth it

**Review by cthulhu-rider — APPROVED** [review]

File: pkg/local_object_storage/metabase/VERSION.md [hunk 2]
```diff
@@ -93,6 +93,10 @@ Numbers stand for a single byte value.
   - Name: container ID + `16`
   - Key: split ID
   - Value: list of object IDs
+- Bucket mapping first object ID to a list of objects IDs
+  - Name: container ID + `19`
+  - Key: first object ID
+  - Value: objects for corresponding split chain
```
Comment (roman-khimov) [inline pkg/local_object_storage/metabase/VERSION.md:100]: https://github.com/nspcc-dev/neofs-node/issues/2808

File: pkg/local_object_storage/metabase/VERSION.md (hunk 1 above)
Comment (roman-khimov) [reply pkg/local_object_storage/metabase/VERSION.md:8]: Yeah, they will get out of sync quickly.

**Review by roman-khimov — APPROVED** [review]

File: pkg/local_object_storage/metabase/VERSION.md (hunk 1 above)
Comment (carpawell) [reply pkg/local_object_storage/metabase/VERSION.md:8]: i do not like this file in general. can you suggest something?

**Pull request merged**